src/app/core/models/safetyStatus.model.ts

```diff
@@ -5,5 +5,5 @@ export interface SafetyStatusModel {
   sectionInfo: string;
   statusPowerProduction: Status;
   statusCoreTemperature: Status;
-  extendedStatus?: ReactorModel;
```

Comment:
Did you check rest of the code to be sure that after removal of this property app won't brake